core/src/main/java/nl/nn/adapterframework/pipes/UnzipPipe.java

```diff
@@ -327,4 +330,14 @@ public void setCreateSubdirectories(boolean b) {
 	public boolean isCreateSubdirectories() {
 		return createSubdirectories;
 	}
+	
+	@IbisDoc({"if set <code>true</code>, validation on directory is ignored", "false"})
+	public boolean isTestDirectory()
+	{
+		return testDirectory;
+	}
+	public void setTestDirectory(boolean testDirectory)
```

Comment:
Test directory sounds like it has something to do with testing, which can be a little bit confusing. Try and think more in the direction of checkDirectory or createDirectory(-IfNotExist)

Reply:
I think I get where you are coming from. As for the class itself the attribute name is independent of the use case it was created for. Which means e.g. checkDirectory name is a better for the Class itself. 

I changed the attribute name to checkDirectory.